src/compas_timber/design/wall_from_surface.py

```diff
@@ -359,20 +365,29 @@ def offset_elements(self, element_loop):
                 element.offset(self.edge_stud_offset)
             offset_loop.append(element)
             # self.edges.append(Line(element.centerline[0], element.centerline[1]))
+
         for i, element in enumerate(offset_loop):
             if self.edge_stud_offset > 0:
                 if element.type != "plate":
                     element_before = offset_loop[i - 1]
                     element_after = offset_loop[(i + 1) % len(offset_loop)]
-                    start_point = intersection_line_line(element.centerline, element_before.centerline, 0.01)[0]
-                    end_point = intersection_line_line(element.centerline, element_after.centerline, 0.01)[0]
+                    start_point = intersection_line_line(
+                        element.centerline, element_before.centerline, self.dist_tolerance
+                    )[0]
+                    end_point = intersection_line_line(
+                        element.centerline, element_after.centerline, self.dist_tolerance
+                    )[0]
                     if start_point and end_point:
                         element.centerline = Line(start_point, end_point)
+                    else:
+                        raise ValueError("edges are parallel, no intersection found")
             else:
                 element_before = offset_loop[i - 1]
                 element_after = offset_loop[(i + 1) % len(offset_loop)]
-                start_point = intersection_line_line(element.centerline, element_before.centerline, 0.01)[0]
-                end_point = intersection_line_line(element.centerline, element_after.centerline, 0.01)[0]
+                start_point = intersection_line_line(
+                    element.centerline, element_before.centerline, self.dist_tolerance
+                )[0]
+                end_point = intersection_line_line(element.centerline, element_after.centerline, self.dist_tolerance)[0]
```

Comment:
nit pickly and could be personal preference, but I find
```python3
end_point, _ = intersection_line_line(...)
```
more readable and pythonin than
```python3
end_point = intersection_line_line(...)[0]
```

Reply:
changed this as requested